playbooks/matrix-mult.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
name of the playbook: `gpu_operator_run_matrix-mult.yml`

Reply:
Hi @kpouget This test runs in under a minute so I think it could be included in the nightly CI.